ansible/roles/util-cfg-service/templates/terrain.properties.j2

```diff
@@ -99,7 +99,6 @@ terrain.fs.bad-chars                  = {{ irods.bad_chars }}
 terrain.fs.max-paths-in-request       = {{ fs_max_paths_in_request }}
 terrain.fs.kifshare-download-template = {{ kifshare.de_url }}
 terrain.fs.kifshare-external-url      = {{ kifshare_external_url }}
-terrain.fs.anon-user                  = {{ anon_files.anon_user }}
```

Comment:
Since the anon-files endpoint was already migrated, this is no longer used in terrain.